test/helpers/web.test.ts

```diff
@@ -39,11 +54,23 @@ describe('Web Helpers', () => {
 
   it('Can POST to the uploader endpoint (HTTP)', async () => {
     uploadURL = 'http://codecov.io'
-    // deepcode ignore WrongNumberOfArgs/test: believe this is a false positive
-    nock('http://codecov.io')
-      .post('/upload/v4')
-      .query(true)
-      .reply(200, 'testPOSTHTTP')
+    mockClient.intercept({
```

Comment:
```suggestion
    mockClient = mockAgent.get(uploadURL)
    mockClient.intercept({
```

Reply:
Make sure you prime the client for the specific host that requests are being sent to.